kolibri/core/discovery/models.py

```diff
@@ -10,6 +12,14 @@ class NetworkLocation(models.Model):
     which can be used to sync content or data.
     """
 
+    class Meta:
+        ordering = ["added"]
```

Comment:
because this model's ID has been changed to a uuid string, we need a consistent way to sort `NetworkLocation` objects